src/riak_kv_pb_timeseries.erl

```diff
@@ -1,65 +1,65 @@
 -module(riak_kv_pb_timeseries).
 
--include_lib("riak_pb/include/riak_kv_pb.hrl").
--include_lib("riak_ql/include/riak_ql_sql.hrl").
--include_lib("riak_ql/include/riak_ql_ddl.hrl").
--behaviour(riak_api_pb_service).
+% -include_lib("riak_pb/include/riak_kv_pb.hrl").
+% -include_lib("riak_ql/include/riak_ql_sql.hrl").
```

Comment:
riak_ql_sql.hrl has been removed, its contents is now in riak_ql_ddl.hrl (https://github.com/basho/riak_ql/pull/22 merged today).